test/services/katello/registration_manager_test.rb

```diff
@@ -71,6 +71,26 @@ def test_existing_host_mismatch_uuid
           assert @klass.validate_hosts(hosts, @org, @host.name, 'different-uuid')
         end
 
+        def test_host_profile_assume_build_mode_only_not_in_build
+          @host.subscription_facet.update(dmi_uuid: 'existing_system_uuid')
+          Setting[:host_profile_assume] = true
```

Comment:
Since we've established this feature is works outside the bounds of host_profile_assume, this should be changed to `false` for both tests

Reply:
I believe I've made the requested changes.